src/components/StyleGuide/Iconography.stories.mdx

```diff
@@ -111,33 +108,31 @@ The graphic of the `Icon` component is handled through the `IconNames` and `Logo
 | `LogoNames.logo_nypl`          |
 | `LogoNames.logo_brooklyn`      |
 
-### Size
+## Icon Sizes
 
 The size of the `Icon` component is handled through the `IconSizes` enum.
 
 <Preview>
-  <Story name="Size">
-    <div style={{ display: "flex", alignItems: "center" }}>
-      <span style={{ marginRight: "1rem" }}>
-        <Icon decorative size={IconSizes.small} name="check" />
-      </span>
-      <span style={{ marginRight: "1rem" }}>
-        <Icon decorative size={IconSizes.medium} name="check" />
-      </span>
-      <span style={{ marginRight: "1rem" }}>
-        <Icon decorative size={IconSizes.large} name="check" />
-      </span>
-      <span style={{ marginRight: "1rem" }}>
-        <Icon decorative size={IconSizes.xlarge} name="check" />
-      </span>
-      <span style={{ marginRight: "1rem" }}>
-        <Icon decorative size={IconSizes.xxlarge} name="check" />
-      </span>
-      <span style={{ marginRight: "1rem" }}>
-        <Icon decorative size={IconSizes.xxxlarge} name="check" />
-      </span>
-    </div>
-  </Story>
+  <div style={{ display: "flex", alignItems: "center" }}>
```

Comment:
Is it possible to use the `Stack` component from Chakra for this layout?

Reply:
Sure.  Let me give it a try.

Reply:
Actually, I will wait on this for now and wait until Chakra is integrated into `development`.